### PR TITLE
Added section about Content Fragments workflows

### PR DESCRIPTION
**Pull request by naydav**
Result https://developer-stage.adobe.com/uix/docs/extension-manager/extension-developed-by-adobe/content-fragments-workflows/

### CODE REVIEW

File: src/pages/extension-manager/extension-developed-by-adobe/content-fragments-workflows/index.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+# Content Fragments workflows
+
+This extension provides UI extensibility, allowing customers to initiate an AEM workflow for selected content fragments.
+
+##  Extension overview
+
+The extension enhances two AEM services: [AEM Content Fragments Console](../../../services/aem-cf-console-admin/) and [AEM Content Fragment Editor](../../../services/aem-cf-editor/). It adds a button to the `Action Bar` for the Content Fragments Console and a button to the `Header Menu` for the AEM Content Fragment Editor.
+
+![AEM Content Fragments Console](workflow-extension-cf-admin.png)
+
```
Comment (naydav) [inline src/pages/extension-manager/extension-developed-by-adobe/content-fragments-workflows/index.md:17]: Need to add some "spacing"

File: src/pages/extension-manager/extension-developed-by-adobe/content-fragments-workflows/index.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+- `Start Workflow button` - when clicked, initiates the workflow initialization process. The workflow starts asynchronously in the background of the application.
+
+To launch a workflow, you need to fill in all the required fields and click the "Start Workflow" button. This will initiate the workflow for each content fragment asynchronously. Each content fragment will have its own workflow process.
+
+## Working with references
+
+Within the grid listing the content fragments, there are two additional columns: `References` and `References By` along with the ability to mark these options for each content fragment in the list using checkboxes.
+
+- `References` - by selecting this checkbox, you indicate that a separate workflow process should be created for all children fragments (references) up to the maximum allowed depth. If no children fragments (references) exist for this content fragment, the checkbox will not be displayed.
+- `References By` - by selecting this checkbox, you indicate that a process will be created for its parent reference as well. If the parent fragment's reference does not exist for this content fragment, the checkbox will not be displayed.
```
[the remark this answers is not in the document]
Comment (naydav) [reply src/pages/extension-manager/extension-developed-by-adobe/content-fragments-workflows/index.md]: done

**Review by fe-lix- — APPROVED** [review]

**Review by lenaorobei — APPROVED** [review]

File: src/pages/extension-manager/extension-developed-by-adobe/content-fragments-workflows/index.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
+---
+title: Content Fragments workflows
+description: Explore enhanced functionality in the Content Fragments Console and AEM Content Fragment Editor, simplifying workflow initiation.
+contributors:
+- AdobeDocs/uix
+---
+
+# Content Fragments workflows
+
+This extension provides UI extensibility, allowing customers to initiate an AEM workflow for selected content fragments.
```
Comment (kokoc) [inline src/pages/extension-manager/extension-developed-by-adobe/content-fragments-workflows/index.md:10]: I think the extension doesn't really provide UI extensibility. It mostly uses it

File: src/pages/extension-manager/extension-developed-by-adobe/content-fragments-workflows/index.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+##  Extension overview
+
+The extension enhances two AEM services: [AEM Content Fragments Console](../../../services/aem-cf-console-admin/) and [AEM Content Fragment Editor](../../../services/aem-cf-editor/). It adds a button to the `Action Bar` for the Content Fragments Console and a button to the `Header Menu` for the AEM Content Fragment Editor.
```
Comment (kokoc) [inline src/pages/extension-manager/extension-developed-by-adobe/content-fragments-workflows/index.md:14]: This sentence sounds a little strange to me. Would be nice to understand what exactly this button does from the beginning. I suggest moving an information from the last sentence in this paragraph somewhere higher.

File: src/pages/extension-manager/extension-developed-by-adobe/content-fragments-workflows/index.md (hunk 3 above)
Comment (naydav) [reply src/pages/extension-manager/extension-developed-by-adobe/content-fragments-workflows/index.md]: Rephrased

File: src/pages/extension-manager/extension-developed-by-adobe/content-fragments-workflows/index.md (hunk 4 above)
Comment (naydav) [reply src/pages/extension-manager/extension-developed-by-adobe/content-fragments-workflows/index.md:14]: Moved the above sentence up.
`When clicked, it opens a modal dialog through which you can initiate a workflow.`

**Pull request merged**